src/ert/libres_facade.py

```diff
@@ -413,15 +413,15 @@ def load_all_summary_data(
                 raise IndexError(f"No such realization {realization_index}")
             realizations = [realization_index]
 
-        summary_keys = self.get_summary_keys()
-        if keys is not None:
+        summary_keys = list(fs.getSummaryKeySet().keys())
+        if keys:
             summary_keys = [
                 key for key in keys if key in summary_keys
             ]  # ignore keys that doesn't exist
-
+        summary_keys.sort()
```

Comment:
Why are we sorting the keys now?

Reply:
They were sorted in `ensemble_config`, but not in the key list from storage.

Reply:
Alright, just a little worried that this is now an O(n*logn) when it used to be O(n), but should be small compared to actually fetching the data 🤔

Reply:
I see your consern, the number of keys are somewhat limited though, so dont think it will have a large impact (famous last words)